working/augmentation-libraries/feature-specification.md

```diff
@@ -599,40 +597,41 @@ An augmenting enum value is allowed to invoke a different constructor than
 the original enum value, or provide an argument list where none was present
 before.
 
-New enum values may also be defined in the augmenting library, and they will
-be appended to the original values in source and augmentation traversal order.
+New enum values may also be defined in the augmentation, and they will be
+appended to the original values in source and augmentation traversal order.
```

Comment:
(I'd try to define it so that there is only one order, which is source order within a file, then recursive traverseal of the augmentation files of `augment` directives in the current file, in their source order. Then this is just "in augmentation traversal order", the source order within a file is included in that. Otherwise pedants might start asking what the priority is between "source" and "augmentation traversal" order. They do that.)

Reply:
I changed to just "in augmentation traversal order", that is what you meant right? :)